bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -3663,7 +3663,7 @@ public ServerConfiguration setHttpServerTrustStorePath(String trustStorePath) {
      * @return http server trustStore password
      */
     public String getHttpServerTrustStorePassword() {
-        return getString(HTTP_SERVER_KEY_STORE_PASSWORD);
+        return getString(HTTP_SERVER_TRUST_STORE_PASSWORD);
```

Comment:
should we handle retrocompatibility here? the risk is that the user upgrades and the webserver doesn't work anymore

at least, we should add highlight in the release notes

Reply:
Please add fallback to the old property. Also a comment in the code pointing to this pull would be valuable